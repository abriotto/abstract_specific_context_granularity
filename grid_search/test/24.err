  0%|                                                                                                                                                                                                    | 0/99 [00:00<?, ?it/s]  6%|###########3                                                                                                                                                                                | 6/99 [00:00<00:01, 54.57it/s] 17%|################################1                                                                                                                                                          | 17/99 [00:00<00:01, 81.29it/s] 26%|#################################################1                                                                                                                                         | 26/99 [00:00<00:01, 68.08it/s] 34%|################################################################2                                                                                                                          | 34/99 [00:00<00:00, 66.39it/s] 41%|#############################################################################4                                                                                                             | 41/99 [00:00<00:00, 62.45it/s] 48%|##########################################################################################6                                                                                                | 48/99 [00:00<00:00, 56.97it/s] 58%|###########################################################################################################6                                                                               | 57/99 [00:00<00:00, 64.03it/s] 65%|########################################################################################################################8                                                                  | 64/99 [00:01<00:00, 59.06it/s] 72%|######################################################################################################################################1                                                    | 71/99 [00:01<00:00, 57.97it/s] 82%|#########################################################################################################################################################                                  | 81/99 [00:01<00:00, 64.59it/s] 91%|##########################################################################################################################################################################                 | 90/99 [00:01<00:00, 69.29it/s] 99%|#########################################################################################################################################################################################1 | 98/99 [00:01<00:00, 70.27it/s]100%|###########################################################################################################################################################################################| 99/99 [00:01<00:00, 65.35it/s]
  0%|                                                                                                                                                                                                    | 0/25 [00:00<?, ?it/s] 28%|####################################################6                                                                                                                                       | 7/25 [00:00<00:00, 63.65it/s] 56%|########################################################################################################7                                                                                  | 14/25 [00:00<00:00, 58.58it/s] 88%|####################################################################################################################################################################5                      | 22/25 [00:00<00:00, 64.68it/s]100%|###########################################################################################################################################################################################| 25/25 [00:00<00:00, 63.56it/s]
