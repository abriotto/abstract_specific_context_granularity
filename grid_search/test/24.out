# ["--attributes=3", "--values=4", "--game_size=3", "--vocab_size_factor=3", "--n_epochs=100", "--batch_size=64", "--learning_rate=0.0005", "--hidden_size=128", "--temp_update=0.97", "--temperature=1"]
Creating train_ds and val_ds...

Creating test_ds...
{"loss": 0.6972362995147705, "acc": 0.48764461278915405, "length": 3.6369807720184326, "mode": "train", "epoch": 1}
{"loss": 0.690737247467041, "acc": 0.5292202830314636, "length": 3.3701658248901367, "mode": "test", "epoch": 1}
{"loss": 0.6928431987762451, "acc": 0.5081987380981445, "length": 3.5995376110076904, "mode": "train", "epoch": 2}
{"loss": 0.6875640153884888, "acc": 0.5666540265083313, "length": 1.5690608024597168, "mode": "test", "epoch": 2}
{"loss": 0.6938761472702026, "acc": 0.5051066279411316, "length": 3.5794739723205566, "mode": "train", "epoch": 3}
{"loss": 0.6901962161064148, "acc": 0.5476981401443481, "length": 1.1381216049194336, "mode": "test", "epoch": 3}
{"loss": 0.6916638016700745, "acc": 0.5094783306121826, "length": 3.463348150253296, "mode": "train", "epoch": 4}
{"loss": 0.6872036457061768, "acc": 0.5674605369567871, "length": 1.2762430906295776, "mode": "test", "epoch": 4}
{"loss": 0.6901271939277649, "acc": 0.5302544236183167, "length": 3.3775124549865723, "mode": "train", "epoch": 5}
{"loss": 0.6818400621414185, "acc": 0.5950397253036499, "length": 1.7016574144363403, "mode": "test", "epoch": 5}
{"loss": 0.6869856715202332, "acc": 0.5404470562934875, "length": 3.2796170711517334, "mode": "train", "epoch": 6}
{"loss": 0.6750556826591492, "acc": 0.6446313261985779, "length": 1.9723756313323975, "mode": "test", "epoch": 6}
{"loss": 0.6810336709022522, "acc": 0.579284131526947, "length": 2.963426113128662, "mode": "train", "epoch": 7}
{"loss": 0.6662354469299316, "acc": 0.6605209708213806, "length": 2.19889497756958, "mode": "test", "epoch": 7}
{"loss": 0.6624628901481628, "acc": 0.6477015614509583, "length": 2.8122622966766357, "mode": "train", "epoch": 8}
{"loss": 0.6475605964660645, "acc": 0.6878734827041626, "length": 2.7624309062957764, "mode": "test", "epoch": 8}
{"loss": 0.6398299336433411, "acc": 0.6739562749862671, "length": 2.9568569660186768, "mode": "train", "epoch": 9}
{"loss": 0.6266374588012695, "acc": 0.670242965221405, "length": 2.7071824073791504, "mode": "test", "epoch": 9}
{"loss": 0.6037572622299194, "acc": 0.6943961381912231, "length": 2.867823839187622, "mode": "train", "epoch": 10}
{"loss": 0.600883960723877, "acc": 0.6934139132499695, "length": 3.1823203563690186, "mode": "test", "epoch": 10}
{"loss": 0.5698620676994324, "acc": 0.7096520066261292, "length": 3.307600498199463, "mode": "train", "epoch": 11}
{"loss": 0.5916385650634766, "acc": 0.6979166865348816, "length": 3.988950252532959, "mode": "test", "epoch": 11}
{"loss": 0.5460600852966309, "acc": 0.7261934280395508, "length": 3.6741421222686768, "mode": "train", "epoch": 12}
{"loss": 0.5868315696716309, "acc": 0.7062615156173706, "length": 4.0, "mode": "test", "epoch": 12}
{"loss": 0.5216578245162964, "acc": 0.7405803203582764, "length": 3.8943183422088623, "mode": "train", "epoch": 13}
{"loss": 0.5760500431060791, "acc": 0.7071823477745056, "length": 4.0, "mode": "test", "epoch": 13}
{"loss": 0.497619092464447, "acc": 0.7537491917610168, "length": 3.9461562633514404, "mode": "train", "epoch": 14}
{"loss": 0.5548129081726074, "acc": 0.7384899854660034, "length": 4.0, "mode": "test", "epoch": 14}
{"loss": 0.47210603952407837, "acc": 0.7690308094024658, "length": 3.9500575065612793, "mode": "train", "epoch": 15}
{"loss": 0.5655499696731567, "acc": 0.7467771768569946, "length": 4.0, "mode": "test", "epoch": 15}
{"loss": 0.4457932114601135, "acc": 0.7958856225013733, "length": 3.970881223678589, "mode": "train", "epoch": 16}
{"loss": 0.5504676103591919, "acc": 0.7458563446998596, "length": 4.0, "mode": "test", "epoch": 16}
{"loss": 0.4339216947555542, "acc": 0.7914572358131409, "length": 3.9637272357940674, "mode": "train", "epoch": 17}
{"loss": 0.5374550819396973, "acc": 0.7753223180770874, "length": 4.0, "mode": "test", "epoch": 17}
{"loss": 0.4231269657611847, "acc": 0.8095005750656128, "length": 3.9676942825317383, "mode": "train", "epoch": 18}
{"loss": 0.5472815036773682, "acc": 0.7826887965202332, "length": 4.0, "mode": "test", "epoch": 18}
{"loss": 0.4196292459964752, "acc": 0.8132827877998352, "length": 3.9852166175842285, "mode": "train", "epoch": 19}
{"loss": 0.5248798727989197, "acc": 0.8011049628257751, "length": 4.0, "mode": "test", "epoch": 19}
{"loss": 0.41448092460632324, "acc": 0.8204417824745178, "length": 3.975651502609253, "mode": "train", "epoch": 20}
{"loss": 0.5348474383354187, "acc": 0.7826887965202332, "length": 4.0, "mode": "test", "epoch": 20}
{"loss": 0.40927213430404663, "acc": 0.8247714638710022, "length": 3.9896717071533203, "mode": "train", "epoch": 21}
{"loss": 0.5024039149284363, "acc": 0.8020257353782654, "length": 4.0, "mode": "test", "epoch": 21}
{"loss": 0.40500813722610474, "acc": 0.8259260058403015, "length": 3.980956554412842, "mode": "train", "epoch": 22}
{"loss": 0.5338555574417114, "acc": 0.7817679643630981, "length": 4.0, "mode": "test", "epoch": 22}
{"loss": 0.40294232964515686, "acc": 0.825154721736908, "length": 3.9895362854003906, "mode": "train", "epoch": 23}
{"loss": 0.5161641836166382, "acc": 0.7836095690727234, "length": 4.0, "mode": "test", "epoch": 23}
{"loss": 0.3955332338809967, "acc": 0.8322173953056335, "length": 3.9816904067993164, "mode": "train", "epoch": 24}
{"loss": 0.5112177133560181, "acc": 0.7918969392776489, "length": 4.0, "mode": "test", "epoch": 24}
{"loss": 0.39454546570777893, "acc": 0.833153486251831, "length": 3.99006986618042, "mode": "train", "epoch": 25}
{"loss": 0.5072247982025146, "acc": 0.7928177714347839, "length": 4.0, "mode": "test", "epoch": 25}
{"loss": 0.39295345544815063, "acc": 0.8307313323020935, "length": 3.9881749153137207, "mode": "train", "epoch": 26}
{"loss": 0.4963403642177582, "acc": 0.7983425259590149, "length": 4.0, "mode": "test", "epoch": 26}
{"loss": 0.3853429853916168, "acc": 0.8381500244140625, "length": 3.9917173385620117, "mode": "train", "epoch": 27}
{"loss": 0.5040470361709595, "acc": 0.8066298961639404, "length": 4.0, "mode": "test", "epoch": 27}
{"loss": 0.38411644101142883, "acc": 0.842674732208252, "length": 3.9905712604522705, "mode": "train", "epoch": 28}
{"loss": 0.5136059522628784, "acc": 0.8038674592971802, "length": 4.0, "mode": "test", "epoch": 28}
{"loss": 0.372909277677536, "acc": 0.8402805924415588, "length": 3.9913015365600586, "mode": "train", "epoch": 29}
{"loss": 0.5039586424827576, "acc": 0.8112338781356812, "length": 4.0, "mode": "test", "epoch": 29}
{"loss": 0.38164839148521423, "acc": 0.8420771360397339, "length": 3.991586446762085, "mode": "train", "epoch": 30}
{"loss": 0.4913172423839569, "acc": 0.8112338781356812, "length": 4.0, "mode": "test", "epoch": 30}
{"loss": 0.38293084502220154, "acc": 0.8375341892242432, "length": 3.991971015930176, "mode": "train", "epoch": 31}
{"loss": 0.48941919207572937, "acc": 0.8093921542167664, "length": 4.0, "mode": "test", "epoch": 31}
{"loss": 0.3733961880207062, "acc": 0.843887209892273, "length": 3.9873147010803223, "mode": "train", "epoch": 32}
{"loss": 0.49628567695617676, "acc": 0.8149171471595764, "length": 4.0, "mode": "test", "epoch": 32}
{"loss": 0.36950212717056274, "acc": 0.8473227024078369, "length": 3.988405466079712, "mode": "train", "epoch": 33}
{"loss": 0.46962058544158936, "acc": 0.8167586922645569, "length": 4.0, "mode": "test", "epoch": 33}
{"loss": 0.3623952865600586, "acc": 0.8523882627487183, "length": 3.9951462745666504, "mode": "train", "epoch": 34}
{"loss": 0.4701779782772064, "acc": 0.8222835659980774, "length": 4.0, "mode": "test", "epoch": 34}
{"loss": 0.3533729016780853, "acc": 0.8517055511474609, "length": 3.9936745166778564, "mode": "train", "epoch": 35}
{"loss": 0.46303948760032654, "acc": 0.8195212483406067, "length": 4.0, "mode": "test", "epoch": 35}
{"loss": 0.35112157464027405, "acc": 0.8557355403900146, "length": 3.9892261028289795, "mode": "train", "epoch": 36}
{"loss": 0.4711143970489502, "acc": 0.8176796436309814, "length": 4.0, "mode": "test", "epoch": 36}
{"loss": 0.34662580490112305, "acc": 0.8629114031791687, "length": 3.9962472915649414, "mode": "train", "epoch": 37}
{"loss": 0.47675690054893494, "acc": 0.814917266368866, "length": 4.0, "mode": "test", "epoch": 37}
{"loss": 0.3481518626213074, "acc": 0.8604954481124878, "length": 3.996777296066284, "mode": "train", "epoch": 38}
{"loss": 0.47294896841049194, "acc": 0.8213626742362976, "length": 4.0, "mode": "test", "epoch": 38}
{"loss": 0.3329443633556366, "acc": 0.8573682308197021, "length": 3.997114658355713, "mode": "train", "epoch": 39}
{"loss": 0.4559842050075531, "acc": 0.8268876075744629, "length": 4.0, "mode": "test", "epoch": 39}
{"loss": 0.3382779657840729, "acc": 0.8577194809913635, "length": 3.998011589050293, "mode": "train", "epoch": 40}
{"loss": 0.461065411567688, "acc": 0.826887845993042, "length": 4.0, "mode": "test", "epoch": 40}
{"loss": 0.33266788721084595, "acc": 0.8614779114723206, "length": 3.9871416091918945, "mode": "train", "epoch": 41}
{"loss": 0.4583345055580139, "acc": 0.8268876671791077, "length": 4.0, "mode": "test", "epoch": 41}
{"loss": 0.3286587595939636, "acc": 0.8617058396339417, "length": 3.9961154460906982, "mode": "train", "epoch": 42}
{"loss": 0.4448562562465668, "acc": 0.8296502232551575, "length": 4.0, "mode": "test", "epoch": 42}
{"loss": 0.33078324794769287, "acc": 0.8622308969497681, "length": 3.9921658039093018, "mode": "train", "epoch": 43}
{"loss": 0.44040974974632263, "acc": 0.8259668350219727, "length": 4.0, "mode": "test", "epoch": 43}
{"loss": 0.3260590136051178, "acc": 0.8629968166351318, "length": 3.99786114692688, "mode": "train", "epoch": 44}
{"loss": 0.44511139392852783, "acc": 0.8268876671791077, "length": 4.0, "mode": "test", "epoch": 44}
{"loss": 0.32685697078704834, "acc": 0.8676314949989319, "length": 3.9984662532806396, "mode": "train", "epoch": 45}
{"loss": 0.44848597049713135, "acc": 0.8305708765983582, "length": 4.0, "mode": "test", "epoch": 45}
{"loss": 0.3170787990093231, "acc": 0.8675710558891296, "length": 3.9947357177734375, "mode": "train", "epoch": 46}
{"loss": 0.4458286762237549, "acc": 0.826887845993042, "length": 4.0, "mode": "test", "epoch": 46}
{"loss": 0.3218082785606384, "acc": 0.8728821277618408, "length": 3.998739004135132, "mode": "train", "epoch": 47}
{"loss": 0.4394408166408539, "acc": 0.8370165824890137, "length": 4.0, "mode": "test", "epoch": 47}
{"loss": 0.30486229062080383, "acc": 0.8778822422027588, "length": 3.996721029281616, "mode": "train", "epoch": 48}
{"loss": 0.43469101190567017, "acc": 0.8278083801269531, "length": 4.0, "mode": "test", "epoch": 48}
{"loss": 0.31851181387901306, "acc": 0.8716206550598145, "length": 3.998204469680786, "mode": "train", "epoch": 49}
{"loss": 0.4326275587081909, "acc": 0.8370165824890137, "length": 4.0, "mode": "test", "epoch": 49}
{"loss": 0.2912052869796753, "acc": 0.8809825778007507, "length": 3.9925615787506104, "mode": "train", "epoch": 50}
{"loss": 0.43140849471092224, "acc": 0.8360957503318787, "length": 4.0, "mode": "test", "epoch": 50}
{"loss": 0.290956974029541, "acc": 0.8843269348144531, "length": 3.9991891384124756, "mode": "train", "epoch": 51}
{"loss": 0.4292837083339691, "acc": 0.837016761302948, "length": 4.0, "mode": "test", "epoch": 51}
{"loss": 0.29064035415649414, "acc": 0.8815184235572815, "length": 3.9984328746795654, "mode": "train", "epoch": 52}
{"loss": 0.41722190380096436, "acc": 0.8388582468032837, "length": 4.0, "mode": "test", "epoch": 52}
{"loss": 0.2841812074184418, "acc": 0.8833557963371277, "length": 3.9977478981018066, "mode": "train", "epoch": 53}
{"loss": 0.43332168459892273, "acc": 0.8333333134651184, "length": 4.0, "mode": "test", "epoch": 53}
{"loss": 0.3121035099029541, "acc": 0.875952422618866, "length": 3.99725604057312, "mode": "train", "epoch": 54}
{"loss": 0.4362955689430237, "acc": 0.841620683670044, "length": 4.0, "mode": "test", "epoch": 54}
{"loss": 0.28704774379730225, "acc": 0.8839054107666016, "length": 3.9958808422088623, "mode": "train", "epoch": 55}
{"loss": 0.4115760922431946, "acc": 0.8379373550415039, "length": 4.0, "mode": "test", "epoch": 55}
{"loss": 0.2786077558994293, "acc": 0.8787347078323364, "length": 3.996013641357422, "mode": "train", "epoch": 56}
{"loss": 0.41067707538604736, "acc": 0.8499080538749695, "length": 4.0, "mode": "test", "epoch": 56}
{"loss": 0.2750376760959625, "acc": 0.8892622590065002, "length": 3.997704029083252, "mode": "train", "epoch": 57}
{"loss": 0.4027189612388611, "acc": 0.8434621691703796, "length": 4.0, "mode": "test", "epoch": 57}
{"loss": 0.2909054160118103, "acc": 0.8883467316627502, "length": 3.999255895614624, "mode": "train", "epoch": 58}
{"loss": 0.413469523191452, "acc": 0.8443830013275146, "length": 4.0, "mode": "test", "epoch": 58}
{"loss": 0.27390041947364807, "acc": 0.8885237574577332, "length": 3.9966745376586914, "mode": "train", "epoch": 59}
{"loss": 0.401248037815094, "acc": 0.8517494201660156, "length": 4.0, "mode": "test", "epoch": 59}
{"loss": 0.2650439441204071, "acc": 0.8946999311447144, "length": 3.9947965145111084, "mode": "train", "epoch": 60}
{"loss": 0.4040343463420868, "acc": 0.8508287072181702, "length": 4.0, "mode": "test", "epoch": 60}
{"loss": 0.26034247875213623, "acc": 0.8942373394966125, "length": 3.9996798038482666, "mode": "train", "epoch": 61}
{"loss": 0.40176448225975037, "acc": 0.8480662703514099, "length": 4.0, "mode": "test", "epoch": 61}
{"loss": 0.2667495906352997, "acc": 0.8972842693328857, "length": 3.9987878799438477, "mode": "train", "epoch": 62}
{"loss": 0.40138548612594604, "acc": 0.8443831205368042, "length": 4.0, "mode": "test", "epoch": 62}
{"loss": 0.25793710350990295, "acc": 0.8982619643211365, "length": 3.999835968017578, "mode": "train", "epoch": 63}
{"loss": 0.39539480209350586, "acc": 0.8443831205368042, "length": 4.0, "mode": "test", "epoch": 63}
{"loss": 0.25349655747413635, "acc": 0.8994773030281067, "length": 3.999236822128296, "mode": "train", "epoch": 64}
{"loss": 0.39117345213890076, "acc": 0.8572743535041809, "length": 4.0, "mode": "test", "epoch": 64}
{"loss": 0.25053682923316956, "acc": 0.900418221950531, "length": 3.99969744682312, "mode": "train", "epoch": 65}
{"loss": 0.39832088351249695, "acc": 0.8480662703514099, "length": 4.0, "mode": "test", "epoch": 65}
{"loss": 0.2603651285171509, "acc": 0.8985297679901123, "length": 3.9988200664520264, "mode": "train", "epoch": 66}
{"loss": 0.3871611952781677, "acc": 0.8535911440849304, "length": 4.0, "mode": "test", "epoch": 66}
{"loss": 0.2549664080142975, "acc": 0.9028369784355164, "length": 3.9984238147735596, "mode": "train", "epoch": 67}
{"loss": 0.3883570432662964, "acc": 0.8563537001609802, "length": 4.0, "mode": "test", "epoch": 67}
{"loss": 0.24530740082263947, "acc": 0.8998897671699524, "length": 3.9938406944274902, "mode": "train", "epoch": 68}
{"loss": 0.3863935172557831, "acc": 0.854512095451355, "length": 4.0, "mode": "test", "epoch": 68}
{"loss": 0.24230337142944336, "acc": 0.9006547927856445, "length": 3.997833013534546, "mode": "train", "epoch": 69}
{"loss": 0.37000077962875366, "acc": 0.8591161966323853, "length": 4.0, "mode": "test", "epoch": 69}
{"loss": 0.2437036782503128, "acc": 0.9059931635856628, "length": 3.999891757965088, "mode": "train", "epoch": 70}
{"loss": 0.3821917474269867, "acc": 0.8572743535041809, "length": 4.0, "mode": "test", "epoch": 70}
{"loss": 0.24359746277332306, "acc": 0.9002788662910461, "length": 3.9965121746063232, "mode": "train", "epoch": 71}
{"loss": 0.3736717104911804, "acc": 0.8609576225280762, "length": 4.0, "mode": "test", "epoch": 71}
{"loss": 0.24166710674762726, "acc": 0.9054547548294067, "length": 3.9949069023132324, "mode": "train", "epoch": 72}
{"loss": 0.3809337317943573, "acc": 0.8664825558662415, "length": 4.0, "mode": "test", "epoch": 72}
{"loss": 0.24510914087295532, "acc": 0.9064088463783264, "length": 3.9959981441497803, "mode": "train", "epoch": 73}
{"loss": 0.38557130098342896, "acc": 0.8609576225280762, "length": 4.0, "mode": "test", "epoch": 73}
{"loss": 0.25327226519584656, "acc": 0.9009783267974854, "length": 3.9978203773498535, "mode": "train", "epoch": 74}
{"loss": 0.403583288192749, "acc": 0.8535911440849304, "length": 4.0, "mode": "test", "epoch": 74}
{"loss": 0.24072475731372833, "acc": 0.9041186571121216, "length": 3.999105453491211, "mode": "train", "epoch": 75}
{"loss": 0.39101237058639526, "acc": 0.8499079942703247, "length": 4.0, "mode": "test", "epoch": 75}
{"loss": 0.24302199482917786, "acc": 0.9050406813621521, "length": 3.999379873275757, "mode": "train", "epoch": 76}
{"loss": 0.38696154952049255, "acc": 0.8563535809516907, "length": 4.0, "mode": "test", "epoch": 76}
{"loss": 0.23941665887832642, "acc": 0.9053000211715698, "length": 3.998403787612915, "mode": "train", "epoch": 77}
{"loss": 0.38971078395843506, "acc": 0.8471455574035645, "length": 4.0, "mode": "test", "epoch": 77}
{"loss": 0.24982333183288574, "acc": 0.905231237411499, "length": 3.9973974227905273, "mode": "train", "epoch": 78}
{"loss": 0.3773311376571655, "acc": 0.8572744131088257, "length": 4.0, "mode": "test", "epoch": 78}
{"loss": 0.23440761864185333, "acc": 0.9090809226036072, "length": 3.9997622966766357, "mode": "train", "epoch": 79}
{"loss": 0.3800310492515564, "acc": 0.8572745323181152, "length": 4.0, "mode": "test", "epoch": 79}
{"loss": 0.22336329519748688, "acc": 0.9112423062324524, "length": 3.9996960163116455, "mode": "train", "epoch": 80}
{"loss": 0.3898974061012268, "acc": 0.8618784546852112, "length": 4.0, "mode": "test", "epoch": 80}
{"loss": 0.22628739476203918, "acc": 0.91429203748703, "length": 3.998692274093628, "mode": "train", "epoch": 81}
{"loss": 0.37924930453300476, "acc": 0.8609576225280762, "length": 4.0, "mode": "test", "epoch": 81}
{"loss": 0.22412678599357605, "acc": 0.9105782508850098, "length": 3.9988386631011963, "mode": "train", "epoch": 82}
{"loss": 0.3914518356323242, "acc": 0.8572744131088257, "length": 4.0, "mode": "test", "epoch": 82}
{"loss": 0.22321762144565582, "acc": 0.9168134331703186, "length": 3.9998772144317627, "mode": "train", "epoch": 83}
{"loss": 0.38446903228759766, "acc": 0.8609576225280762, "length": 4.0, "mode": "test", "epoch": 83}
{"loss": 0.2209266573190689, "acc": 0.9163990616798401, "length": 3.9971513748168945, "mode": "train", "epoch": 84}
{"loss": 0.41390371322631836, "acc": 0.8563535213470459, "length": 4.0, "mode": "test", "epoch": 84}
{"loss": 0.2102930098772049, "acc": 0.9168112277984619, "length": 3.9997942447662354, "mode": "train", "epoch": 85}
{"loss": 0.39606428146362305, "acc": 0.8572744131088257, "length": 4.0, "mode": "test", "epoch": 85}
{"loss": 0.21582812070846558, "acc": 0.9146503210067749, "length": 3.999699592590332, "mode": "train", "epoch": 86}
{"loss": 0.399771511554718, "acc": 0.8627992272377014, "length": 4.0, "mode": "test", "epoch": 86}
{"loss": 0.21045823395252228, "acc": 0.9129372835159302, "length": 3.996467351913452, "mode": "train", "epoch": 87}
{"loss": 0.3952246606349945, "acc": 0.8554328083992004, "length": 4.0, "mode": "test", "epoch": 87}
{"loss": 0.22217054665088654, "acc": 0.9186269640922546, "length": 3.998971462249756, "mode": "train", "epoch": 88}
{"loss": 0.4077720046043396, "acc": 0.8517495393753052, "length": 4.0, "mode": "test", "epoch": 88}
{"loss": 0.22704876959323883, "acc": 0.9207274317741394, "length": 3.998356342315674, "mode": "train", "epoch": 89}
{"loss": 0.39689821004867554, "acc": 0.8637200593948364, "length": 4.0, "mode": "test", "epoch": 89}
{"loss": 0.22371827065944672, "acc": 0.9105090498924255, "length": 3.9976320266723633, "mode": "train", "epoch": 90}
{"loss": 0.38938698172569275, "acc": 0.8591160178184509, "length": 4.0, "mode": "test", "epoch": 90}
{"loss": 0.22015011310577393, "acc": 0.913683295249939, "length": 3.9988811016082764, "mode": "train", "epoch": 91}
{"loss": 0.4061654806137085, "acc": 0.8572745323181152, "length": 4.0, "mode": "test", "epoch": 91}
{"loss": 0.22433771193027496, "acc": 0.9140333533287048, "length": 3.9998767375946045, "mode": "train", "epoch": 92}
{"loss": 0.4028703570365906, "acc": 0.8581951260566711, "length": 4.0, "mode": "test", "epoch": 92}
{"loss": 0.21139362454414368, "acc": 0.9127787947654724, "length": 3.9996144771575928, "mode": "train", "epoch": 93}
{"loss": 0.39153581857681274, "acc": 0.8618784546852112, "length": 4.0, "mode": "test", "epoch": 93}
{"loss": 0.21625009179115295, "acc": 0.9241561889648438, "length": 3.998180866241455, "mode": "train", "epoch": 94}
{"loss": 0.3970434069633484, "acc": 0.8683242201805115, "length": 4.0, "mode": "test", "epoch": 94}
{"loss": 0.19862842559814453, "acc": 0.9211158752441406, "length": 3.9991774559020996, "mode": "train", "epoch": 95}
{"loss": 0.4029940962791443, "acc": 0.862799346446991, "length": 4.0, "mode": "test", "epoch": 95}
{"loss": 0.2048875391483307, "acc": 0.920219898223877, "length": 3.9999501705169678, "mode": "train", "epoch": 96}
{"loss": 0.40085792541503906, "acc": 0.8646409511566162, "length": 4.0, "mode": "test", "epoch": 96}
{"loss": 0.19507578015327454, "acc": 0.9257776141166687, "length": 3.999818801879883, "mode": "train", "epoch": 97}
{"loss": 0.3998033404350281, "acc": 0.8581951856613159, "length": 4.0, "mode": "test", "epoch": 97}
{"loss": 0.2202499806880951, "acc": 0.9180523157119751, "length": 3.9998981952667236, "mode": "train", "epoch": 98}
{"loss": 0.41771817207336426, "acc": 0.8499079942703247, "length": 4.0, "mode": "test", "epoch": 98}
{"loss": 0.21382743120193481, "acc": 0.9124168157577515, "length": 3.9977927207946777, "mode": "train", "epoch": 99}
{"loss": 0.42950141429901123, "acc": 0.8489871621131897, "length": 4.0, "mode": "test", "epoch": 99}
{"loss": 0.20330148935317993, "acc": 0.9152739644050598, "length": 3.9999773502349854, "mode": "train", "epoch": 100}
{"loss": 0.4293205440044403, "acc": 0.8480662703514099, "length": 4.0, "mode": "test", "epoch": 100}
test accuracy: 0.7583334445953369
